### ISSUE TITLE
Training issue after Django Integration

### ISSUE THREAD

**Issue by anantbara** [gunthercox/ChatterBot#840]
Getting issue in training with below chatterbot settings in settings.py

``` Python
# Chatbot Settings
CHATTERBOT = {
    'name': 'Vyabl Bot',
    'storage_adapter':'chatterbot.storage.MongoDatabaseAdapter',
    'database':'vyablbot_db',
    'database_uri':'mongodb://127.0.0.1:27017',
    'logic_adapters': [{
                                        'import_path': 'chatterbot.logic.BestMatch',
                                        'statement_comparison_function': 'chatterbot.comparisons.levenshtein_distance',
                                        'response_selection_method': 'chatterbot.response_selection.get_most_frequent_response'
                                    },
                                     {
                                        'import_path': 'chatterbot.logic.LowConfidenceAdapter',
                                        'threshold': 0.6,
                                        'default_response': 'I am sorry, but I do not understand.'
                                    },],
    'filters':['chatterbot.filters.RepetitiveResponseFilter'],
    'trainer': 'chatterbot.trainers.ChatterBotCorpusTrainer',
    'training_data': [
            'D:\\Anant\\Anant\\ChatBot\\ChatBot_3_Chatterbot\\ChatbotDjango-new\\VyablBot\\Data\\Greetings',
            'D:\\Anant\\Anant\\ChatBot\\ChatBot_3_Chatterbot\\ChatbotDjango-new\\VyablBot\\Data\\FAQ_Json',
            'D:\\Anant\\Anant\\ChatBot\\ChatBot_3_Chatterbot\\ChatbotDjango-new\\VyablBot\\Data\\FAQ_Dataset_1',
			
    ],
    'django_app_name': 'django_chatterbot',
    'use_django_models':False,
}
```

*Error generated*
![error](https://user-images.githubusercontent.com/24452345/28198400-88aec676-687b-11e7-8ef5-027d42af8c74.jpg)

**Comment by anantbara** [comment]
Having issue in logic adapter ('import_path')

**Comment by vkosuri** [comment]
Similar issue https://github.com/gunthercox/ChatterBot/issues/717.

@anantbara might be this solution will work https://github.com/gunthercox/ChatterBot/issues/717#issuecomment-309488888

**Comment by gunthercox** [comment]
Fixed by #1174
